nixos/modules/virtualisation/containers.nix

```diff
@@ -112,10 +112,16 @@ in
 
   config = lib.mkIf cfg.enable {
 
+    environment.systemPackages = [ pkgs.oci-seccomp-bpf-hook ];
```

Comment:
Does this need to be added to `systemPackages` or can it go in the wrapper?

Reply:
Ah I think we do not need it at all because the hook already specifies the full path to the binary.